pkg/env/costmodelenv.go

```diff
@@ -89,6 +91,18 @@ func IsEmitPodAnnotationsMetric() bool {
 	return GetBool(EmitPodAnnotationsMetricEnvVar, false)
 }
 
+// IsEmitKubeNodeStatusCapacityMemoryBytesMetric returns true if cost-model is configured
+// to emit the kube_node_status_capacity_memory_bytes metric.
+func IsEmitKubeNodeStatusCapacityMemoryBytesMetric() bool {
+	return GetBool(EmitKubeNodeStatusCapacityMemoryBytesMetricEnvVar, false)
```

Comment:
Do we want these defaulting to `true` or `false`? I see some conversation about this in https://github.com/kubecost/cost-analyzer-helm-chart/pull/926.

Reply:
I don't mind the decision at all. Just looking for (1) acknowledgement that this is intentional (not a bug) and preferably (2) a justification for maintaining opposite defaults.

Reply:
I went for false in the code because I figured that this should really be a feature to _enable_ in the actual binary/container while it should be explicitly _disabled_ in the standard deployment of the product. I'm thinking this way because, long-term, if anyone is going to roll our code independently (i.e. package and run the OSS themselves) they are probably going to be running KSM v2 from now on and wouldn't expect us to start emitting a KSM v1 metric.

However, on further consideration, I think this actually should default to `true` here as well because all of the code still expects `kube_node_status_capacity_memory_bytes` and `kube_node_status_capacity_cpu_cores` even if it's running in an environment with only KSM v2 metrics. Just changed it.